.sass-lint.yml

```diff
@@ -0,0 +1,103 @@
+#########################
+## Sample Sass Lint File
+#########################
+# Linter Options
+options:
+  # Set the formatter to 'html'
+  formatter: stylish
+# File Options
+files:
+  include: 'src/**/*.scss'
+# Rule Configuration
+rules:
+  # Extends
+  extends-before-mixins: 2
+  extends-before-declarations: 2
+  placeholder-in-extend: 2
+
+  # Mixins
+  mixins-before-declarations:
+    - 2
+    -
+      exclude:
+        - 'respond-at-and-above'
+        - 'respond-between'
+        - 'respond-below'
+        - 'after'
+        - 'before'
+
+  # Line Spacing
+  one-declaration-per-line: 2
+  empty-line-between-blocks: 2
+  single-line-per-selector: 2
+
+  # Disallows
+  no-color-keywords: 2
+  no-color-literals: 0
+  no-css-comments: 0
+  no-debug: 2
+  no-duplicate-properties: 2
+  no-empty-rulesets: 2
+  no-extends: 0
+  no-ids: 2
+  no-important: 2
+  no-invalid-hex: 2
+  no-mergeable-selectors:
+    - 2
+    -
+      whitelist:
+        - 'from'
+        - 'to'
+  no-misspelled-properties: 2
+  no-qualifying-elements: 2
+  no-trailing-zero: 2
+  no-transition-all: 2
```

Comment:
Not sure, actually. But I like the idea of forcing the developer to judiciously add transitions so that we don't get carried away with transitioning all when it's not actually necessary:

```
transition: color 1s, box-shadow 1s;
color: ...;
box-shadow: ...;
border: ...; // transition not required on border
font-weight: ...; // transition not required on font-weight
```